src/collective/mustread/interfaces.py

```diff
@@ -190,10 +192,91 @@ def who_unread(obj, force_deadline=True):
         If ``force_deadline`` is False, returns scheduled users who
         did not read the object at all.
 
+        @guido: i'd suggest to remove force_deadline as we have the get_stats
+        method to get a detailed report which users read and when
+
         :param obj: Object that should be read by users
         :type obj: Content object (must be IUUID resolvable)
         :param force_deadline: Whether to ignore reads after the deadline
         :type force_deadline: Bool
-        :returns: Userids of users scheduled for reading which did not read
-        :rtype: List
+        :returns: Dictionary with userids of users scheduled for reading which
+                  did not read and their deadline (``{userid: deadline}``)
+        :rtype: dict
+        '''
+
+    def what_to_read(context=None, userid=None):
+        '''Query which content items have open read requests.
+
+        This only queries for objects who have been explicitly scheduled via
+        the ``schedule_must_read()`` method.
+        If no read requests have been scheduled an empty list is returned.
+
+        If context is given, limit the result to objects located within
+        this context. If context is ``None`` search within the plonesite.
+
+        If userid is given, limit results to objects with an open read request
+        for this userid.
+
+        @guido: if we implement the force_deadline here, we'd have an easy way
+        to list all items with missed deadlines too. alternatively we can make
+        get_report more powerful/complex
+
+        :param context: Context to search for objects with open read requests
+        :type context: Content object (must be IUUID resolvable)
+        :param userid: Userid to search for open read requests for.
+        :type userid: string
+        :returns: List of content objects with open read requests
+        :rtype: Generator
+        '''
+        # xxx guido: generator or list? (we intend to use all elements anyway)
```

Comment:
taken... as what_to_read returns only a few objects (compared to the report) i'll make it return a list and stay with the generator for the report